MicrosoftTeams.schema.json

```diff
@@ -802,7 +802,7 @@
                         "maxLength": 2048
                       }
                     },
-                    "supportsAnonymousAccess": {
+                    "supportsAnonymizedPayloads": {
```

Comment:
Wouldnt this break existing apps?

Weren't we supposed to have both of these side-by-side in devPreview for a bit?